autoload/ctrlp/mrufiles.vim

```diff
@@ -56,16 +57,22 @@ fu! s:reformat(mrufs, ...)
 endf
 
 fu! s:record(bufnr)
-	if s:exclnomod && &l:modifiable | retu | en
-  if s:exclnomod && !&l:modifiable | retu | en
```

Comment:
Hey sorry, I totally forgot about this. So anyway, it looks to me that it would have been early-exiting either way and never get on to the rest of the function, which is why I'm confused. What am I missing?